### DISCUSSION TITLE
Optionally disable shipping and billing address on Cart.

### DISCUSSION THREAD

**Discussion by kris914g** [lunarphp/lunar#1045] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I have tried to implement lunar for my webshop, which sells digital products that are sent by email. Therefore, it is not necessary to add a shipping address. However, it is necessary in relation to the cart's createOrder method, which is not appropriate in the context of digital products.

**Describe the solution you'd like**
It would be good if it could be possible to disable the necessity for the shipping and billing address (maybe except state, postcode, and country_id), and possibly set the country_id in the backend so the customer doesn't need to fill any input fields regarding street name, etc. Thereby it is still possible to use the tax system.

**Comment by csiber** [comment]
Hi. Did you find any solution for this? I would have the same problem.

**Comment by alecritson** [comment]
Digital products are certainly something on our radar but not planned as of yet. In the meantime you could customise the pipelines in the config files, so you replace the actions and handle how addresses work for your webshop manually and without the need for the user to input anything.
